Dockerfile

```diff
@@ -64,6 +64,9 @@ RUN curl -sS -L https://github.com/facebook/buck/releases/download/v${BUCK_VERSI
 
 # Full reference at https://dl.google.com/android/repository/repository2-1.xml
 # download and unpack android
+RUN echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
```

Comment:
Can you join those two lines in one ?

Reply:
Also, the command looks not working

```
sysctl: setting key "fs.inotify.max_user_watches": Read-only file system
The command '/bin/sh -c echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p' returned a non-zero code: 255
```

Reply:
that only works on the host OS fought that same battle before. 
Whatever is running docker engine should have that done.

Temporary:
```bash
sudo sysctl fs.file-max=2097152
sudo sysctl fs.inotify.max_user_watches=524288
sudo sysctl -p
```

Permanent
```bash
echo 'fs.file-max=2097152' | sudo tee -a /etc/sysctl.conf
echo 'fs.inotify.max_user_watches=524288' | sudo tee -a /etc/sysctl.conf
sudo sysctl -p
```

Try the above on where docker is installed and rerun should then work.